spyder/plugins/configdialog.py

```diff
@@ -398,7 +401,13 @@ def save_to_conf(self):
         for radiobutton, (option, _default) in list(self.radiobuttons.items()):
             self.set_option(option, radiobutton.isChecked())
         for lineedit, (option, _default) in list(self.lineedits.items()):
-            self.set_option(option, to_text_string(lineedit.text()))
+            option_value = lineedit.text()
+            old_value = self.get_option(option, _default)
+            if isinstance(old_value, list):
+                option_value = [int(e) for e in option_value.split(',')]

```

Comment:
I don't know if this is a good decision. What would happen here if another option is a list of strings?